dev/io.openliberty.cdi.core_fat/bnd.bnd

```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src
+
+tested.features:\
+    cdi-2.0,\
```

Comment:
what about cdi-1.2?

Reply:
I saw you mentioned EE7 later.

Reply:
Features which are listed directly in the `server.xml` do not also get listed in `tested.features`.

Reply:
I figured out when I saw the last file.